_posts/2017/06/2017-06-05-SteeringCommitteeRetreat.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+authors: ["The 2017 Steering Committee"]
+title: "Summary of the 2017 Software Carpentry Steering Committee Retreat "
+date: 2017-05-31
+time: "10:00:00"
+category: ["Steering Committee", "Community", "Instructor Training"]
+---
+
+The Software Carpentry Steering Committee met in person on June 7-8 in Davis, California.   The committee members have unique journeys that brought them through the community to having a seat on the steering committee. Importantly, we have a unified vision for the future of the Software Carpentry Foundation and the roles of the steering committee. Below is a brief summary of the motions passes and detailed description of the five key areas over which the Steering Committee will continue to provide oversight. 
+ 
+Below, we 1) summarize the motions that were passed at the meeting, 2) describe the five key areas over which the Steering Committee provides oversight, and 3) provide a timeline for implementing our goals.
+
+## Summary of motions passed at retreat
+
+- The Steering Committee has created a board-designated operating reserve ($80,000, equivalent to one quarter’s worth of budget) to provide an assurance of financial solvency
+- The Steering Committee empowers the trainer group to move forward with open instructor training events to non-member affiliated individuals 
+- The Steering Committee empowers staff to perform assigned tasks, with the Steering Committee retaining oversight in the key areas of community, instructor training, curriculum, finances, and staff
```

Comment:
Might I suggest "empowers staff to perform tasks in their areas of focus" . . . "and hiring staff"